### ISSUE TITLE
White line in none resizable window

### ISSUE THREAD

**Issue by teawithfruit** [trueinteractions/tint2#100]
How is it possible to remove the big white line in the bottom of none resizable windows?
![White line](https://www.trueinteractions.com/tint2/docs/images/window-resizable0_yosemite_0_window.png)

**Comment by trevorlinton** [comment]
@teawithfruit Sorry for the late response, we delved deep into this issue with no luck until I thought i'd try the same thing (making a window non-resizable) in Xcode/objective-c.  To my surprise regular apps built with objective-c have this exact same issue on yosemite (see screenshot below).  I've filed a bug with Apple to see if this is a rendering artifact bug or some really bizarre "feature".

This does not occur if you use a non-textured window, e.g., set window.texture = false, and the rendering artifact goes away.

Hope this helps,
Trevor

![screen shot 2015-02-26 at 6 26 37 pm](https://cloud.githubusercontent.com/assets/3111873/6405804/62baadce-bde5-11e4-8200-0616c335acae.png)
